study.md

```diff
@@ -108,7 +120,18 @@ boomhauer.hello
 Write some JavaScript code that duplicates this functionality.
 
 ```javascript
-// your answer here
```

Comment:
const Person = function(name) {
  this.name = name;
}

Person.prototype.hello = function() {
  console.log(`Hello, my name is ${this.name}.`);
}

const boomhauer = new Person('Jeff Boomhauer');

boomhauer.hello();